test/e2e/common/tektonconfigdeployment_test.go

```diff
@@ -291,6 +291,55 @@ func runAddonTest(t *testing.T, clients *utils.Clients, tc *v1alpha1.TektonConfi
 			t.Errorf("Addon params in TektonConfig not equal to TektonAddon params: %s", diff.PrintWantGot(d))
 		}
 	})
+
+	t.Run("validate-addon-params", func(t *testing.T) {
+
+		addonsIS, err := clients.Operator.TektonInstallerSets().List(context.TODO(), metav1.ListOptions{
+			LabelSelector: "operator.tekton.dev/created-by=TektonAddon",
+		})
+		if err != nil {
+			t.Fatalf("failed to get InstallerSet: %v", err)
+		}
+
+		// there must be 4 installerSet created for addons
```

Comment:
based on code it should be 5 :thinking: 
```suggestion
		// there must be 5 installerSet created for addons
```

Reply:
@sm43 its good to merge :+1: just above comment fix